src/mono/msbuild/apple/build/AppleBuild.props

```diff
@@ -9,6 +9,24 @@
     <MonoForceInterpreter Condition="'$(MonoForceInterpreter)' == ''">true</MonoForceInterpreter>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(PublishReadyToRun)' == 'true' and '$(UseMonoRuntime)' != 'true'">
+    <PublishReadyToRunComposite>true</PublishReadyToRunComposite>
+    <PublishReadyToRunContainerFormat>macho</PublishReadyToRunContainerFormat>
+  </PropertyGroup>
+
+  <ItemGroup Condition="'$(PublishReadyToRun)' != 'true' and '$(UseMonoRuntime)' != 'true'">
+    <EnvironmentVariables Include="DOTNET_ReadyToRun=0" />
```

Comment:
Do we actually need to set this? Isn't having no R2R data enough?

Reply:
Ok, I will do it in https://github.com/dotnet/runtime/pull/121305.